ui/app/src/components/ContentTypeManagement/components/EditTypeView.tsx

```
@@ -0,0 +1,527 @@
+							</Typography>
+						</Box>
+					</>
+				}
+			/>
+			{
+				// region TODO: Temp Dialog to show XML
+				<Dialog open={Boolean(openXmlViewer)} onClose={() => setOpenXmlViewer(null)} maxWidth="lg" fullWidth>
+					<Typography
+						variant="body2"
+						component="pre"
+						dangerouslySetInnerHTML={{ __html: openXmlViewer }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)